### PR TITLE
Put in text styles, fix doffing video, add welfare screen

### PR DESCRIPTION
**Pull request by adamkoch**
Adds new typography styles #21 
Adds initial welfare screen #30 (although still needs proper icons)
Fixes doffing video file #53 

## Screenshots

Screen Shot 2020-03-24 at 7 56 10 am

### CODE REVIEW

**Review by ericjiang97 — APPROVED** [review]
Lgtm! 🔥 🚀

File: pubspec.yaml [hunk 1]
```diff
@@ -34,5 +35,17 @@ flutter:
   fonts:
     - family: Inter
       fonts:
-        - asset: assets/fonts/Inter.ttf
+        - asset: assets/fonts/Inter-300-Light.ttf
```
Comment (domesticmouse) [inline pubspec.yaml:38]: nit: consider using package:google_fonts instead of hard including inter.

More detail: https://pub.dev/packages/google_fonts

**Review by domesticmouse — APPROVED** [review]

File: pubspec.yaml (hunk 1 above)
Comment (adamkoch) [reply pubspec.yaml:38]: Downside of google fonts is it requires http download one time before fonts load. Not a fan of that.

Comment (domesticmouse) [reply pubspec.yaml:38]: Fair enough. Have you raised a bug on `google_fonts` about the http download requirement? I'll happily advocate for a build time option with the eng team.

Comment (adamkoch) [reply pubspec.yaml:38]: I haven't raised it. I assumed it was WAI as it gives you easy access to the online library of Google fonts (major plus) with the downside it needs to do a small download upon startup. Using local assets seems to be a fine alternative if you just want a small number of fonts with no http download (downside being increased binary size).

Comment (domesticmouse) [reply pubspec.yaml:38]: Fair. I'll still bug the team tho =)

**Pull request merged**